### DISCUSSION TITLE
Multiple Kratos Instances with same database.

### DISCUSSION THREAD

**Discussion by vsumit89** [ory/kratos#2991] [Q&A] [answered]
I have 2 apps A1 and A2, I want them to have independent registration/login screens. So for that I am deploying 2 instances of kratos but I want both of instances to store the data in the same database so that when a user who registered on app A1 can login on A2 without having to register on A2 ( since I am storing identities in the same database). Is it possible ?

**Comment by vinckr** [comment, marked as answer]
Hello @vsumit89,
should be possible yes. 
I think you need to have your identity schema in sync (so the same for both instances), but otherwise I dont see a big issue.

Let us know how it goes!

**Comment by Untel** [comment]
What about having a single Kratos instance but customs pages depending on the referer ?
(I don't know if it's possible, just asking for curiosity)
